components/MobileNavigation.js

```diff
@@ -148,10 +176,10 @@ export default class MobileNavigation extends Component {
   }
 }
 
-function NavigationLink ({ href, disabled, children }) {
+function NavigationLink ({ href, disabled, children, active }) {
   return (
-    <div className={disabled ? 'disabled' : ''}>
-      <LevitatingLink href={href}>{children}</LevitatingLink>
+    <div className={active ? 'disabled' : ''}>
```

Comment:
The class name should represent what it’s doing!

Now `disabled` doesn’t work anymore...

The correct class name should look like this:

| disabled | active | result className |
| --- | --- | --- |
| `false` | `false` | `''` |
| `false` | `true` | `'active'` |
| `true` | `false` | `'disabled'` |
| `true` | `true` | `'disabled active'` |

You can do it like this:

```js
  const className = [
    disabled && 'disabled',
    active && 'active'
  ].filter(name => name).join(' ')

  return (
    <div className={className}>
      ...
```